### PR TITLE
Simple linked list: Remove spammy link

### PR DESCRIPTION
**Pull request by treble37**
It looks like this site is no longer maintained. If you visit it in your
browser you get what appears to be a spammy site.

### CODE REVIEW

File: exercises/simple-linked-list/README.md [hunk 1]
```diff
@@ -131,7 +131,7 @@ If you want to know more about Exercism, take a look at the [contribution guide]
 
 ## Source
 
-Inspired by 'Data Structures and Algorithms with Object-Oriented Design Patterns in Ruby', singly linked-lists. [http://www.brpreiss.com/books/opus8/html/page96.html#SECTION004300000000000000000](http://www.brpreiss.com/books/opus8/html/page96.html#SECTION004300000000000000000)
+Inspired by 'Data Structures and Algorithms with Object-Oriented Design Patterns in Ruby', singly linked-lists.
```
Comment (coriolinus) [inline exercises/simple-linked-list/README.md:134]:
```suggestion
Inspired by '[Data Structures and Algorithms with Object-Oriented Design Patterns in Ruby](https://web.archive.org/web/20160414064110/http://www.brpreiss.com/books/opus8/)', singly linked-lists.
```

**Comment by treble37** [comment]
Thanks for the tip - PR for metadata here https://github.com/exercism/problem-specifications/pull/1648

**Comment by coriolinus** [comment]
Hmm, CI is still complaining after a restart. It is almost certainly a whitespace issue, or something similar.

If you download configlet `bin/fetch-configlet` and possibly also the problem specifications repo, you should be able to do `configlet generate . --only simple-linked-list`, or a command to that effect. That should regenerate the README into a format that CI is happy with.

**Comment by coriolinus** [comment]
Thanks for dealing with all this process! 

As this is a documentation-only change and CI is green, I'm going to omit the ordinary delay for other reviewer comments, and just merge this immediately.

**Comment by treble37** [comment]
no worries, when I first started, I thought "this will be an easy fix" lol :grinning:

**Pull request merged**